src/PactNet.Abstractions/Verifier/IPactVerifierSource.cs

```diff
@@ -7,13 +7,14 @@ namespace PactNet.Verifier
     /// Configured pact verifier source
     /// </summary>
     public interface IPactVerifierSource
-    {
+    {
         /// <summary>
         /// Set up the provider state setup URL so the service can configure states
         /// </summary>
-        /// <param name="providerStateUri">Provider state setup URI</param>
+        /// <param name="providerStateUri">Provider state setup URI</param>
+        /// <param name="teardown">Whether to invoke a teardown request to the provider state endpoint after each interaction</param>
         /// <returns>Fluent builder</returns>
-        IPactVerifierSource WithProviderStateUrl(Uri providerStateUri);
+        IPactVerifierSource WithProviderStateUrl(Uri providerStateUri, bool teardown = false);
```

Comment:
This is a breaking API change, when though it's optional, so needs to be a separate overload instead of an optional parameter